src/package/pywinrt/projection/AzurePipelinesTemplates/job-build-projection.yml

```diff
@@ -10,32 +10,32 @@ jobs:
       Release_x86_py37:
         buildArchitecture: 'x86'
         buildConfiguration: 'Release'
-        pythonVersion: '3.7.x'
```

Comment:
It's not clear to me why these were changed, but then again, I'm not an expert in this space.

What effect does this have, e.g., finding version 3.7.0 vs 3.7.1? I'm assuming you've tested it and it works as expected. Just trying to educate myself.

Reply:
find_package in cmake doesn't use the .x syntax, but the YAML task requires .x. So I removed it from the variable value and added it back in where it's needed in the steps file.

Reply:
Also, in find_package, "3.7 EXACT" will match 3.7.*

Reply:
Yeah, the find_package EXACT bit was the one I was wondering about. I suspected it worked that way, but I feel unsafe making assumptions about CMake. :)